rdtools/normalization.py

```diff
@@ -620,6 +620,9 @@ def _aggregate(time_series, target_frequency, max_timedelta, series_type):
     max_interval_nanoseconds = max_timedelta.total_seconds() * 10.0**9
 
     gap_mask = t_diffs > max_interval_nanoseconds
+    if time_series.index[0] != union_index[0]:
```

Comment:
would using `.sum(min_count=1)` in line 605 above fix the bug? Might offer a cleaner fix.

Reply:
I retract this suggestion